docs/source/projects/architecture/content-architecture.rst

```diff
@@ -102,6 +104,7 @@ to the notebook server, which saves it on disk as a JSON file with a
 ``.ipynb`` extension.
 
 .. image:: figs/notebook_components.png
+   :alt: 
```

Comment:
```suggestion
   :alt: Communication diagram showing five nodes labeled user, browser, notebook server, notebook file, and kernel. The browser, notebook file and kernel cannot talk to each other directly. They communicate indirectly through the notebook server. The user can only talk to the browser. The notebook server and kernel are colored green whereas the browser is colored purple to draw a distinction between the Jupyter front end versus back end.
```

Reply:
Per Ely's suggestion, maybe we should set do blank alt-text: alt=""

Reply:
Refer to [W3C's alt Decision Tree](https://www.w3.org/WAI/tutorials/images/decision-tree/)